channels/ru.m3u

```diff
@@ -101,32 +101,16 @@ https://rt-news.secure.footprint.net/1103.m3u8
-#EXTINF:-1 tvg-id="RTAmerica.ru" tvg-name="RT America" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/tyXIHlj.png" group-title="News",RT America (720p) [Offline]
-http://rt-usa.secure.footprint.net/1105.m3u8
-#EXTINF:-1 tvg-id="RTAmerica.ru" tvg-name="RT America" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/tyXIHlj.png" group-title="News",RT America (720p) [Offline]
-https://rt-news-gd.secure2.footprint.net/1103.m3u8
-#EXTINF:-1 tvg-id="RTAmerica.ru" tvg-name="RT America" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/tyXIHlj.png" group-title="News",RT America (720p) [Offline]
-https://rt-usa-gd.secure2.footprint.net/1105.m3u8
 #EXTINF:-1 tvg-id="RTArabic.ru" tvg-name="RT Arabic" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="" group-title="News",RT Arabic (720p) [Offline]
 https://rt-arab-gd.secure2.footprint.net/1104.m3u8
-#EXTINF:-1 tvg-id="RTArabic.ru" tvg-name="RT Arabic" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="" group-title="News",RT Arabic (720p) [Offline]
-https://rt-arab.secure.footprint.net/1104.m3u8
 #EXTINF:-1 tvg-id="RTDocumentary.ru" tvg-name="RT Documentary" tvg-country="RU" tvg-language="English;Russian" tvg-logo="https://i.imgur.com/XJGki3v.png" group-title="Documentary",RT Documentary (720p) [Offline]
 https://rt-doc-gd.secure2.footprint.net/1101.m3u8
-#EXTINF:-1 tvg-id="RTDocumentary.ru" tvg-name="RT Documentary" tvg-country="RU" tvg-language="English;Russian" tvg-logo="https://i.imgur.com/XJGki3v.png" group-title="Documentary",RT Documentary (720p) [Offline]
-https://rt-doc.secure.footprint.net/1101.m3u8
 #EXTINF:-1 tvg-id="RTenEspanol.ru" tvg-name="RT en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="News",RT en Español (720p) [Offline]
 https://rt-esp-gd.secure2.footprint.net/1102.m3u8
-#EXTINF:-1 tvg-id="RTenEspanol.ru" tvg-name="RT en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="News",RT en Español (720p) [Offline]
-https://rt-esp.secure.footprint.net/1102.m3u8
 #EXTINF:-1 tvg-id="RTFrance.ru" tvg-name="RT France" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/gVedXRh.png" group-title="News",RT France (720p) [Offline]
 https://rt-france-gd.secure2.footprint.net/1107.m3u8
-#EXTINF:-1 tvg-id="RTFrance.ru" tvg-name="RT France" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/gVedXRh.png" group-title="News",RT France (720p) [Offline]
-https://rt-france.secure.footprint.net/1107.m3u8
 #EXTINF:-1 tvg-id="RTUK.ru" tvg-name="RT UK" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/zdTA0ha.png" group-title="News",RT UK (720p) [Offline]
 https://rt-uk-gd.secure2.footprint.net/1106.m3u8
-#EXTINF:-1 tvg-id="RTUK.ru" tvg-name="RT UK" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/zdTA0ha.png" group-title="News",RT UK (720p) [Offline]
-https://rt-uk.secure.footprint.net/1106.m3u8
```

Comment:
UK : https://rt-uk.gcdn.co/live/rtuk/playlist.m3u8 (they've forgot to move their links, managed to get it by deduction)
Arabic (not added to RT's website, needed to play with their audio stream to get it) : https://rt-arb.gcdn.co/live/rtarab/playlist.m3u8
France : https://rt-fra.gcdn.co/live/rtfrance/playlist.m3u8
Global : https://rt-glb.gcdn.co/live/rtnews/playlist.m3u8

Only the Arabic one doesn't seem to be on their GCDN server.